shtub.sh

```diff
@@ -223,16 +223,28 @@ _stub::methods::returns() {
 # Sets a stub to error by printing the given string to stderr and returning the
 # given status code.
 # @param $1 name Name of the command to stub
```

Comment:
I was just using jsdoc style, haha. Not sure there is a good standard for bashdocs... Unless...